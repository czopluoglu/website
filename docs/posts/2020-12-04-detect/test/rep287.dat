01010001100101111111
11011111101010111011
00100000000010001110
01111000101111111111
00111001111111001101
11010011111000100110
10000000000111101011
11111010010010110101
11110001001011001111
11000000011100001100
01100100000010101111
00100001111110111011
00000001011110101111
01000010101001111110
01111111111111001011
10000010001010101110
11001000011110101100
00001110100000000010
11110001001100011011
00110001001100101110
10111001011111111011
00001001111110101110
10101100000110101111
11110011110111101111
11001111101110001101
01011001011110100111
11111010111111010011
10010001100100100111
11110111110001010100
10001001101000000001
01001101011111111111
11111001110000000101
10100110101110111001
11000000010100110010
00111110100110000110
00001011000100101111
11011011111011100000
00000001001010110000
01010111010111011111
01000011010010011111
11111111111111110111
10010011011010110111
01001101101010111010
00010001001000100100
10000101010010110011
10111101011101101001
00110101110110110111
10110110011000000001
00111111011110101110
11011111111011110011
10000000010100110011
11001001001101000111
01100101110110100101
01110001101000111100
11000001000001101010
10011110011100010111
10001101100100101010
11110101100000000110
10001101011100101101
00110110100010010111
11010100100110000010
11001101101111101111
11101001101000110010
01100001001111111110
11011101111110000111
01000001111000101110
10100011011001101100
00101101001001110111
10100000111100111110
10000001011101000010
00001001111010011011
11010011111111110101
10101001111000100110
11000000001010110011
11011101111111111111
10100001111111110111
00001001101110111100
11000100000000101011
00000001010010101010
11110011111100110010
11101011110100111100
10111111111110101001
10101001110000011000
11000011110111100111
10111100110010100110
00010101111000100110
11011010111100001110
01100110111100111011
11111111111000100111
01101001001101110011
01010110001000000100
11110111111110011011
01010001011000100110
10111001101111110111
01010000111111110110
11011111011101101110
00001111100010000001
11001110111010000111
11010101011111011011
00000100101110100111
10010111111000100001
10011101111100011010
00110001001010111100
11011010110100100111
10000001100010111111
10101001111010011111
10100101000000000011
10110000011110100111
00010101101101111111
01100000011111001011
01011011011000000111
11010111111111110011
10011010011100110110
01000001000010100011
11100011101110100100
11110010101010110101
11110101101010011111
00100001001100111111
00000101110010010010
00011111001010100111
11001010001010001010
00001100110000000010
11010111111010110010
01010001111010110110
00100001010000001001
10001101110111001100
00010011111110111110
00001011011000101011
10000101110000100000
00011000001000100100
10000101101111101110
00010100100010101100
11011100101011000100
10000101001010000101
01110010011000010000
11000010111110111110
10001100001001111111
11111111111010111101
11100101011100010111
10001000101011110111
00011000100101110011
01111000110110011011
01001000011011111011
11101111100100111110
11001001001101100111
10001111110111001110
10101001100111110110
01010101010100011100
10001011001011000010
11000100111010111110
01111101111111101111
11111111111111111111
11000011001100101110
11110110111111111101
10001000111000000101
11101100111111111111
01000100101010101101
01100001111011111111
00011001001101001011
00001101110110000111
01001010001011100010
11011101100001011111
01101111111111010111
11001100011100010111
10101001111000110010
00110001001010111111
11100101100111111111
00000100010000000000
00010101001000000001
01100001001010000010
10000001001101000100
10101101101011110111
10011010011010100101
10111101111111111110
00011101111011101101
11100000001110001111
00001001100010110011
11011010011000001101
11000100001100101110
11000001011000001110
01100000001010001011
10110011011011000010
11011100111101111101
11000100000000011011
00011111100011000011
00000001011110101110
00101101011100111001
00101010000110101010
10000111001111001010
10110110011111101111
11001111111100101110
11001111111111010011
11010001001010000011
10010001101000100110
10101001111110001110
10001100001011111001
01101100100111110011
10001000000000000110
11011001011111101111
00000101111111011101
10110100110110101111
10010001011110101111
01111111001111111110
01001000011010110011
11111111110000100111
00010000010011101110
10101011111111111111
01010001001001101111
01001001011110101010
11111001111110111111
10110111001011110110
00101001101010011011
00100000011000000010
01111000111110110101
10011000100110111011
00111100111110010111
10010000011111001110
00000000101010100010
00010010101001000111
01000101001011000010
10110101001011010111
10000101010100100110
10100110101010111110
01011110111000111011
10110100100010010101
10101101101110111111
10001011011011100111
01001100011111000111
10010011101010110111
00000001010000100011
00100100001011101010
11100101010000100010
10100111111010010111
10110010001010000101
00001000111101100100
10111101111011111111
10000101111110101111
11011001010000101010
10001010101100110111
11011101100111000110
10011001100010001110
10000010001010011110
11111100111111111011
10011011000000001011
11000001101000110011
00010011101111001101
01000011011100100101
10110111111110100110
11100111111101001111
11111011010000100101
