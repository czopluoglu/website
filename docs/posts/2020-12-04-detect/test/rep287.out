-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep287.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.00

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.61
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  3
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7445
                   IDN index value:   0.7158
                           Ratio r:   0.6174

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    1    2    1    2    2    1    3    2    1
    1    3    2    3    3    2    3    1    3    2

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    2    4    7   10   11   18

   -----------CLUSTER  2-------------

    3    5    6    9   13   16   20

   -----------CLUSTER  3-------------

    8   12   14   15   17   19
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d--++-+-+++---+-----
   -d++++------+---+-++
   -+d++-++--+--+------
   +++d+++-+++---------
   ++++d+--+-----++-+--
   -+-++d+----++-+-+++-
   +-++-+d---++---++++-
   --+----d-+++++----+-
   +--++---d++--+-++---
   +--+---++d++-+--+---
   +-++--++++d++-+-----
   -----+++-++d++--+-++
   -+---+-+--++d+---+-+
   --+----+++-++d--+---
   +---++----+---d-++++
   ----+-+-+------d++++
   -+---++-++-+-+++d-++
   ----+++-----+-++-d++
   -+---+++---+--++++d+
   -+---------++-+++++d
 
 No cross validation for this DETECT run
